### ISSUE TITLE
Add support for VSCode extension project properties

### ISSUE THREAD

**Issue by unional** [keithamus/sort-package-json#45]
It uses the following:

```js
{
  "engines": {
    "vscode": "^1.25.0"
  },
  "activationEvents": [
    "onCommand:editor.sortPackageJson"
  ],
  "categories": [
    "Other"
  ],
  "contributes": {
    "commands": [
      {
        "command": "editor.sortPackageJson",
        "title": "Sort package.json"
      }
    ]
  },
  "displayName": "Sort package.json",
  "publisher": "unional"
}
```

Should they go before or after `dependencies`?
Should they go before or after `engines`?

I will open a PR once these are decided. 🌷

**Comment by keithamus** [comment]
I'm happy to see this go after `dependencies` but ultimately I'll be happy with whatever you think is sensible.

**Comment by muuvmuuv** [comment]
I use it that way: https://hastebin.com/gazulivebo.json. Would be great to have this autoformatted!

**Comment by keithamus** [comment]
> 🎉 This PR is included in version 1.37.0 🎉
> 
> The release is available on:
> 
>    * [npm package (@latest dist-tag)](https://www.npmjs.com/package/sort-package-json/v/1.37.0)
> 
>    * [GitHub release](https://github.com/keithamus/sort-package-json/releases/tag/v1.37.0)
> 
> 
> Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot 📦🚀
